src/main/java/com/ericsson/eiffel/remrem/publish/service/ResultEvent.java

```diff
@@ -0,0 +1,52 @@
+package com.ericsson.eiffel.remrem.publish.service;
+
+public class ResultEvent {
```

Comment:
I think this class should be renamed. It is not an Event. PublishResult would better reflect what it is.